inviteevent.go

```diff
@@ -0,0 +1,11 @@
+package gomatrixserverlib
+
+// InviteStateFormattedEvent is an event which is explicitly formatted for
+// the 'invite_state' of a /sync response. It is not a real event.
+type InviteStateFormattedEvent struct {
+	Content  rawJSON `json:"content"`
+	RoomID   string  `json:"room_id,omitempty"`
```

Comment:
I don't know, do we? In `/sync` there is room context (upper keys have the room ID), but over federation I think you'll be missing that or something? IDK.